docs/options.md

```diff
@@ -115,6 +115,7 @@ Note that Prettier never unquotes numeric property names in Angular expressions,
 [quote-props-flow]: https://flow.org/try/#0PQKgBAAgZgNg9gdzCYAoVBjOA7AzgFzAA8wBeMAb1TDAAYAuMARlQF8g
 [quote-props-vue]: https://github.com/prettier/prettier/issues/10127
 
+<!-- TODO[@fisker]: Check this -->
 If this option is set to `preserve`, `singleQuote` to `false` (default value), and `parser` to `json5`, double quotes are always used for strings. This effectively allows using the `json5` parser for “JSON with comments and trailing commas”.
```

Comment:
@thorn0 Do you think I can revert #10323?

Reply:
It'd be a breaking change with little gain. Maybe better to keep the logic and just remove this part from the docs.

Reply:
https://github.com/prettier/prettier/pull/15831/commits/ed020b28cacfd48982b313e4a36e5caa042cd664